apps/api/src/app/workflows/workflow.controller.ts

```diff
@@ -212,6 +212,7 @@ export class WorkflowController {
         data: body.data,
         __source: query?.__source,
         type: WorkflowTypeEnum.REGULAR,
+        origin: WorkflowOriginEnum.NOVU,
```

Comment:
remove ferom here

Reply:
same, @djabarovgeorge it cannot be removed. it's the inner command of create we are using as well